eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -248,3 +248,9 @@ services:
         arguments:
          - '%session.save_path%'
          - 'redis'
+
+    ezplatform.core.command.copy_subtree:
+        class: eZ\Bundle\EzPublishCoreBundle\Command\CopySubtreeCommand
+        autowire: true
+        tags:
+            - { name: console.command }
```

Comment:
If we set `autoconfigure: true` do we need to tag it? (just asking, I'm not sure :) )

Reply:
:+1: 
You are right, thought this only works for interfaces.